impeller/display_list/BUILD.gn

```diff
@@ -22,6 +22,13 @@ impeller_component("display_list") {
     "//flutter/fml",
     "//third_party/skia",
   ]
+
+  if (!defined(defines)) {
+    defines = []
+  }
+  if (impeller_enable_3d) {
+    defines += [ "IMPELLER_ENABLE_3D" ]
```

Comment:
Why behind a flag?

Reply:
When I added the stubs originally, I believe there were some lint complaints about unused code.

Reply:
Fuchsia doesn't build the shaders right now, so I ended up guarding everything to avoid the Fuchsia build depending on Impeller shader headers (Nodes depend on Materials, Materials depend on the shader headers).

Reply:
A cleaner solution would probably be to have the DL op and friends use agnostic type (e.g. DlSceneNode) and compile an empty version that errors out on use if 3D support is off.